commons/src/main/java/org/dhis2/commons/bindings/ValueExtensions.kt

```diff
@@ -59,6 +59,23 @@ fun TrackedEntityDataValue?.userFriendlyValue(
     }
 }
 
+fun String.userFriendlyValue(
```

Comment:
is this duplicated in the app? where are you using it?

Reply:
It is part of the app. It just encapsulate the code to be reusable to any string value instead of the TrackedEntityDataValue or TrackendEntityAttributeValue classes in the SDK